CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -518,7 +518,7 @@ public function postProcess() {
         'calendar_month' => 'pledge_calendar_month',
       );
       if ($params['pledge_default_toggle'] == 'contribution_date') {
-        $fieldValue = json_encode(array('contribution_date' => date('Ymd')));
+        $fieldValue = json_encode(array('contribution_date' => date('m/d/Y')));
```

Comment:
Actually, this change was made in order to match how CiviCRM saves dates when used from the datepicker. I will check with Pradeep to see if there are any CRM functions I should be using instead.